lib/http2/stream.c

```diff
@@ -113,20 +113,23 @@ static size_t calc_max_payload_size(h2o_http2_conn_t *conn, h2o_http2_stream_t *
 }
 
 static void encode_data_header_and_consume_window(h2o_http2_conn_t *conn, h2o_http2_stream_t *stream, uint8_t *header,
-                                                  size_t length, int eos)
+                                                  size_t length, h2o_send_state_t send_state, int is_eos)
```

Comment:
I believe `is_eos` can be omitted, since the three states under which this function gets invoked are covered by `send_state`.